### PR TITLE
Fix missing binding resolution for actions referenced by rules.

### LINKED ISSUE #1578: Unable to create a rule using an action belonging to a bound package
When trying to create a rule using an action from a bound package I get an error saying the action does not exist.  I was using a package binding that was bound from a whisk.system package.

### PR DESCRIPTION
**Pull request by rabbah**
Fixes #1578.

### CODE REVIEW

**Comment by rabbah** [comment]
pg1/663 approved.

File: tests/src/system/basic/WskRuleTests.scala [hunk 1]
```diff
@@ -48,7 +48,7 @@ class WskRuleTests
      *
      * @param rules Tuple3s containing (rule, trigger, (actionName, actionFile))
```
Comment (markusthoemmes) [inline tests/src/system/basic/WskRuleTests.scala]: Please update the scaladoc for explanation of the Tuple3 in the Tuple3.

**Comment by cbickel** [comment]
LGTM

**Comment by rabbah** [comment]
thanks for the comments.

**Comment by markusthoemmes** [comment]
rebased, squashed and merged in 2fbb159a3292dcced5aa0e2b31dedd4035eacd30

**Pull request closed without merging**